### PR TITLE
[DOCFIX] Add documentation for remote logging and audit logging

### PR DESCRIPTION
**Pull request by riversand963**
@aaudiber 
I've added the documentation for remote logging and audit logging. Could you take a look? Thanks!

### CODE REVIEW

File: docs/en/Audit-Logging.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Enable Audit Logging
+To enable Alluxio audit logging, you need to set the JVM property. In ./conf/alluxio-env.sh, add the following two lines:
+
+```bash
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logging.enabled=true"
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=${ALLUXIO_MASTER_AUDIT_LOGGER:-MASTER_AUDIT_LOGGER}"
+```
+Then you need to restart Alluxio, and you will find a file named `master_audit.log` in Alluxio's logs directory. The format
+of the file conforms to HDFS's audit log file.
```
Comment (calvinjia) [inline docs/en/Audit-Logging.md]: Could you briefly describe the format, not all users may be familiar with HDFS's format.

File: docs/en/Audit-Logging.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: global
+title: Audit Logging
+group: Features
+priority: 99
```
Comment (calvinjia) [inline docs/en/Audit-Logging.md]: Any reason for making the priority so low?

Comment (riversand963) [reply docs/en/Audit-Logging.md]: No particular reason, just thought it may not be critical.

File: docs/en/Audit-Logging.md (hunk 1 above)
Comment (riversand963) [reply docs/en/Audit-Logging.md]: Done.

File: docs/en/Audit-Logging.md (hunk 2 above)
Comment (riversand963) [reply docs/en/Audit-Logging.md]: Done.

**Comment by aaudiber** [comment]
@riversand963 Since these are two independent documentation pages, could you split this into two separate PRs?

File: docs/en/Audit-Logging.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+title: Audit Logging
+group: Features
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Overview
+Alluxio supports audit logging of user access to your data. With Audit logging, system administrators can
```
Comment (gpang) [inline docs/en/Audit-Logging.md]: NIT: "Audit" should be lower case?

File: docs/en/Audit-Logging.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+keep track of each user's access to data in Alluxio.
+
+## Enable Audit Logging
+To enable Alluxio audit logging, you need to set the JVM property. In ./conf/alluxio-env.sh, add the following two lines:
```
Comment (gpang) [inline docs/en/Audit-Logging.md]: use backticks for `./conf/alluxio-env.sh`

File: docs/en/Audit-Logging.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+```bash
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logging.enabled=true"
+ALLUXIO_MASTER_JAVA_OPTS+=" -Dalluxio.master.audit.logger.type=${ALLUXIO_MASTER_AUDIT_LOGGER:-MASTER_AUDIT_LOGGER}"
+```
+Then you need to restart Alluxio, and you will find a file named `master_audit.log` in Alluxio's logs directory. The format
+of the file is very similar to HDFS's audit log file.
```
Comment (gpang) [inline docs/en/Audit-Logging.md]: This sentence should be with the following table, all in a separate section about the format.

File: docs/en/Remote-Logging.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+ALLUXIO_JAVA_OPTS=" -Dalluxio.logserver.hostname=AlluxioLogServer -Dalluxio.logserver.port=45010"
+ALLUXIO_MASTER_JAVA_OPTS=" -Dalluxio.remote.logger.type=REMOTE_MASTER_LOGGER"
+ALLUXIO_SECONDARY_MASTER_JAVA_OPTS=" -Dalluxio.remote.logger.type=REMOTE_SECONDARY_MASTER_LOGGER"
+ALLUXIO_WORKER_JAVA_OPTS=" -Dalluxio.remote.logger.type=REMOTE_WORKER_LOGGER"
+ALLUXIO_PROXY_JAVA_OPTS=" -Dalluxio.remote.logger.type=REMOTE_PROXY_LOGGER"
+ALLUXIO_LOGSERVER_LOGS_DIR=/tmp/alluxio_remote_logs
+```
+
+## Restart Alluxio And Log Server
+After making the modification to configuration, you need to restart Alluxio and the log server.
```
Comment (gpang) [inline docs/en/Remote-Logging.md]: Do you have to start the log server before Alluxio? If so, that should be explicit.

File: docs/en/Audit-Logging.md (hunk 3 above)
Comment (riversand963) [reply docs/en/Audit-Logging.md]: Will address in another PR.

File: docs/en/Audit-Logging.md (hunk 4 above)
Comment (riversand963) [reply docs/en/Audit-Logging.md]: Will address in another PR.

File: docs/en/Audit-Logging.md (hunk 5 above)
Comment (riversand963) [reply docs/en/Audit-Logging.md]: Will address in another PR.

File: docs/en/Remote-Logging.md (hunk 6 above)
Comment (riversand963) [reply docs/en/Remote-Logging.md]: Will address in another PR.

**Pull request closed without merging**